src/js/game.ts

```diff
@@ -633,6 +633,7 @@ export class Game {
   audioSources: AudioSources;
   sleuth: Sleuths;
   sleuthVoice: SleuthVoices;
+  endGamePhotos: Array<any>;
```

Comment:
Can we be more specific with that type, should it be `Array<HTMLImageElement>` ?

Reply:
This was initially done to fix an error I kept getting, which has since mysteriously disappeared. Updated, and comment added.